exonum-java-binding/core/pom.xml

```diff
@@ -21,9 +21,9 @@
     <project.build.headersDirectory>${project.build.directory}/native-headers</project.build.headersDirectory>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <!-- TODO: stable does not work well until ECR-1839 is resolved. -->
     <rust.compiler.version>stable</rust.compiler.version>
-    <rust.compiler.features>resource-manager</rust.compiler.features>
+    <!-- This should be in sync with features that `integration_tests` crate require from `java_bindings` -->
```

Comment:
Is this wording correct? The library is built here for two purposes:

* to verify it compiles
* to use in Java ITs

The native ITs and the app will build their own versions, if needed.

Reply:
Reworded according to our discussion